src/components/themed/CryptoFiatAmountTile.js

```diff
@@ -32,7 +32,7 @@ export const CryptoFiatAmountTile = (props: Props) => {
   const cryptoAmountDenom = div(nativeCryptoAmount, cryptoDenomMult, DECIMAL_PRECISION)
 
   // Default to 10 displayed chars for crypto amount
-  const fmtCryptoAmount = formatNumber(cryptoAmountDenom, { toFixed: maxCryptoChars === undefined ? MAX_CRYPTO_AMOUNT_CHARACTERS : maxCryptoChars })
+  const fmtCryptoAmount = trimEnd(formatNumber(cryptoAmountDenom, { toFixed: maxCryptoChars === undefined ? MAX_CRYPTO_AMOUNT_CHARACTERS : maxCryptoChars }))
```

Comment:
Needs to trimEnd before formating the number otherwise it would cause a bug in countries that uses ',' instead of '.'